src/components/my-view3.js

```diff
@@ -23,13 +23,13 @@ import { store } from '../store.js';
 import { checkout } from '../actions/shop.js';
 
 // We are lazy loading its reducer.
-import shop from '../reducers/shop.js';
+import shop, { cartQuantitySelector } from '../reducers/shop.js';
```

Comment:
omg what is this syntax. should it be `{shop, cartQuantitySelector}` instead? My brain is imploding

Reply:
This is what it looks like when you import both a module's default export and one of its named exports.

Previously the shop reducer module only exported one thing (the reducer itself), as a default export. When I added named exports for the selectors, I could have removed the default export and instead exported the reducer as a named export, but I didn't do that.

The upside of removing the default and going with only named exports would be less confusing import syntax (per your example). The downside is that this module would be exporting its "main" thing in a different way from the other reducer modules in the template...unless we changed all of them to use named exports as well.

Thoughts? (cc @frankiefu, @keanulee)

Reply:
imo i'd do anything to avoid this syntax lol

Reply:
I don't have a strong opinion on this. But I think this kind of syntax the users need to deal with and learn anyway if you import some other libraries and that libraries may have default and named in the same file.